apps/docs/src/versions.data.ts

```diff
@@ -0,0 +1,43 @@
+import fs from 'fs';
+import path from 'path';
+
+const readVersionsFromFiles = () => {
+  const versions = {
+    FORC: '0.0.0',
+    FUELS: '0.0.0',
+    FUEL_CORE: '0.0.0',
+  };
+
+  try {
+    const forcBinPath = path.join(__dirname, '../../../packages/forc-bin/package.json');
+    const forcFile = fs.readFileSync(forcBinPath, 'utf8');
+    const { config } = JSON.parse(forcFile);
+    ({ forcVersion: versions.FORC } = config);
+
+    const fuelsPath = path.join(__dirname, '../../../packages/fuels/package.json');
+    const fuelsFile = fs.readFileSync(fuelsPath, 'utf8');
+    ({ version: versions.FUELS } = JSON.parse(fuelsFile));
+
+    const dockerfilePath = path.join(__dirname, '../../../services/fuel-core/Dockerfile');
+    const dockerfile = fs.readFileSync(dockerfilePath, 'utf8');
+    const regexFuelcore = /FROM ghcr\.io\/fuellabs\/fuel-core:v(\d+\.\d+\.\d+)/;
+    const match = dockerfile.match(regexFuelcore);
+    versions.FUEL_CORE = match?.[1] || versions.FUEL_CORE;
+  } catch {
+    return versions;
+  }
+
+  return versions;
+};
+
+export default {
+  load() {
+    const { FORC, FUELS, FUEL_CORE } = readVersionsFromFiles();
+
+    return {
+      forc: FORC,
+      fuels: FUELS,
+      fuelCore: FUEL_CORE,
+    };
+  },
+};
```

Comment:
@camsjams Initially, the `versions` package would return `0.0.0` for everything when developing, and then the CI would inject the proper versions during the release. IIRC, this was a problem for the Docs

@Torres-ssf Was it for showing the proper links or something like that? Can't remember it precisely; perhaps I'm confused. Either way, I'll have to refactor the `versions` package to fix a problem that arose after #827. In the end, it might be suitable for such use.

Reply:
Exactly @arboleya . It was made this way for both reasons.

Reply:
Cool, I'll let you know how this pan out.

Reply:
Follow up [issue](https://github.com/FuelLabs/fuels-ts/issues/849) and [PR](https://github.com/FuelLabs/fuels-ts/pull/850).